library/MiscUtils.cpp

```diff
@@ -67,6 +67,8 @@ std::string stl_vsprintf(const char *fmt, va_list args) {
     // Allocate enough memory for the output and null termination
     rv.resize(rsz+1);
     rsz = vsnprintf(&rv[0], rv.size(), fmt, args);
+    if (rsz + 1 < static_cast<int>(rv.size()))
+      rv.resize(std::max(rsz+1,0));
```

Comment:
Z:/home/buildmaster/work/_E69339/_D0/src/library/MiscUtils.cpp(71): error C2589: '(': illegal token on right side of '::'
Z:/home/buildmaster/work/_E69339/_D0/src/library/MiscUtils.cpp(71): error C2059: syntax error: '::'

I need to figure out what causes these error messages.

Reply:
`max` is a macro defined in some Windows header somewhere, I think. There's a definition you can use to disable it. I think we have it set in a few places, but maybe not something `MiscUtils.cpp` includes.

Reply:
```
depends/tthread/fast_mutex.h:  #define NOMINMAX
depends/tthread/tinythread.h:  #define NOMINMAX
library/include/modules/Filesystem.h:    #define NOMINMAX
```